### PR TITLE
Cleaned code to remove all the warnings

### PR DESCRIPTION
**Pull request by Pierre-Demessence**

### CODE REVIEW

File: DiscordBot/Services/AudioService.cs [hunk 1]
```diff
@@ -29,7 +29,7 @@ private async Task ConnectAudioClient()
             _audioClient = await channel.ConnectAsync();
         }
 
-        public async void Music()
+        public async Task Music()
```
Comment (SimplyJpk) [inline DiscordBot/Services/AudioService.cs:32]: Do we still need the AudioService?  Could probably be stripped from the project.

Comment (Pierre-Demessence) [reply DiscordBot/Services/AudioService.cs:27]: We can remove it. Let's remove everything useless.

**Comment by Pierre-Demessence** [comment]
It seems there's a compile error: https://github.com/Unity-Developer-Community/UDC-Bot/pull/20/checks?check_run_id=2565303651

File: DiscordBot/Modules/UserModule.cs [hunk 2]
```diff
@@ -1,231 +1,525 @@
...
+        
+        public UserModule(IServiceProvider serviceProvider, Settings.Deserialized.Settings settings, Rules rules)
         {
-            _loggingService = loggingService;
-            _databaseService = databaseService;
-            _userService = userService;
-            _publisherService = publisherService;
-            _updateService = updateService;
-            _currencyService = currencyService;
+            _databaseService = serviceProvider.GetService<DatabaseService>();
```
Comment (SimplyJpk) [inline DiscordBot/Modules/UserModule.cs:33]: I'm not 100% sold on using serviceProvider here, it works and as far as I can tell doesn't cause any problems but I couldn't find examples of it being used in modules so I'm not sure if a race condition could happen here.

**Comment by SimplyJpk** [comment]
As far as I can tell this is ready to merge into dev. Only !search appears to be broken, and that was broken before all the changes.

**Pull request merged**